include/seqan3/core/algorithm/configuration_element_parallel_mode.hpp

```diff
@@ -0,0 +1,33 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+/*!\file
+ * \brief Provides seqan3::detail::parallel_mode.
+ * \author Svenja Mehringer <svenja.mehringer AT fu-berlin.de>
+ */
+
+#pragma once
+
+#include <seqan3/core/algorithm/pipeable_config_element.hpp>
+
+namespace seqan3::detail
+{
+/*!\brief A global configuration type used to enabled debugging of algorithms.
```

Comment:
`to enabled debugging` what does that mean?

Reply:
ops, copy and paste error from debug mode.
It is supposed to mean that the debug matrices are computed during the alignment.